packages/clay-charts-react/src/Chart.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import BillboardChart from 'react-billboardjs';
+import GeoMap from './GeoMap';
+import Predictive from './Predictive';
+
+import { config } from 'clay-charts-shared';
+
+const {
+	DEFAULT_COLORS,
+	DEFAULT_GRID_OBJECT,
+	DEFAULT_LINE_CLASSES,
+	DEFAULT_POINT_PATTERNS,
+} = config;
+
+export default function(props) {
+	const { color, data, grid, line, point, ...otherProps } = props;
+
+	let ChartComponent;
+
+	switch (data.type) {
+		case 'geo-map':
+			ChartComponent = GeoMap;
+			break;
+		case 'predictive':
+			ChartComponent = Predictive;
+			break;
+		default:
+			ChartComponent = BillboardChart;
```

Comment:
If we want to add any other chart types outside of Billboard, we just add them here